pkg/ring/lifecycler.go

```diff
@@ -97,7 +97,7 @@ type Lifecycler struct {
 
 	// These values are initialised at startup, and never change
 	ID   string
-	addr string
+	Addr string
```

Comment:
Why was this necessary?

Reply:
Addr is used by the ruler to verify ownership, I would rather use ID but that was not in the IngesterDesc.